Realm.Shared/Realm.cs

```diff
@@ -281,64 +292,65 @@ public void Dispose()
         private void Dispose(bool disposing)
         {
             if (IsClosed)
+            {
                 throw new ObjectDisposedException(nameof(Realm));
-            
+            }
+
             if (disposing && !(SharedRealmHandle is UnownedRealmHandle))
             {
                 SharedRealmHandle.CloseRealm();
             }
 
             SharedRealmHandle.Close();  // Note: this closes the *handle*, it does not trigger realm::Realm::close().
 
-            if (stringGetBuffer != IntPtr.Zero)
+            if (StringGetBuffer != IntPtr.Zero)
             {
-                Marshal.FreeHGlobal(stringGetBuffer);
-                stringGetBuffer = IntPtr.Zero;
-                stringGetBufferLen = 0;
+                Marshal.FreeHGlobal(StringGetBuffer);
+                StringGetBuffer = IntPtr.Zero;
+                StringGetBufferLen = 0;
             }
         }
 
-
         /// <summary>
         /// Generic override determines whether the specified <see cref="System.Object"/> is equal to the current Realm.
         /// </summary>
-        /// <param name="rhs">The <see cref="System.Object"/> to compare with the current Realm.</param>
+        /// <param name="obj">The <see cref="System.Object"/> to compare with the current Realm.</param>
         /// <returns><c>true</c> if the Realms are functionally equal.</returns>
-        public override bool Equals(Object rhs)
-        {
-            return Equals(rhs as Realm);
-        }
-
+        public override bool Equals(object obj) => Equals(obj as Realm);
 
         /// <summary>
         /// Determines whether the specified Realm is equal to the current Realm.
         /// </summary>
-        /// <param name="rhs">The Realm to compare with the current Realm.</param>
+        /// <param name="other">The Realm to compare with the current Realm.</param>
         /// <returns><c>true</c> if the Realms are functionally equal.</returns>
-        public  bool Equals(Realm rhs)
+        public bool Equals(Realm other)
```

Comment:
Yes - it's a little annoying that we have to copy-paste the XML docs from Shared to PCL 😞